packages/carousel/src/react/control.tsx

```diff
@@ -34,33 +34,49 @@ const styles = {
     )
 }
 
-export function Control(props) {
+interface ControlProps
+  extends Omit<React.HTMLAttributes<HTMLButtonElement>, 'onClick'> {
+  direction: ValueOf<typeof vars.controlDirections>
+  onClick?: (evt?: React.MouseEvent) => void
+}
+interface ControlStatics {
+  directions: typeof vars.controlDirections
+}
```

Comment:
did you need to replace the `onClick` fn. seems like the types are the same

Reply:
yeah, great point.  Not sure why I did that. Probably just a too-literal conversion from proptypes.

Reply:
Well, doesn't make sense in isolation why I defined line 40.  But when I remove, I get down to combineFns, and I can't harangue Params to be typed correctly.